### PR TITLE
Feature/files

### PR DESCRIPTION
**Pull request by Feudalman**
## Description

Extract system info from project.json into a new folder located in the root of a experiment folder.

|- files
|--- requirements.txt : dependencies in current python.
|--- swanlab-metadata.json : system info such as hardware, python and git.

Closes: #185

### CODE REVIEW

File: swanlab/data/system.py [hunk 1]
```diff
@@ -164,6 +164,10 @@ def get_system_info():
         "gpu": __get_gpu_info(),  # gpu 相关信息
         "git_info": __get_git_branch_and_commit(),  # git 分支和最新 commite 信息
         "command": __get_command(),  # 完整命令行信息
-        "dependencies": __get_pip_requirement(),  # 当前 python 下的依赖信息
         "memory": __get_memory_size(),  # 内存大小
     }
+
+
+def get_requirements() -> str:
```
Comment (SAKURA-CAT) [inline swanlab/data/system.py:154]: 直接把`__get_pip_requirement`改名即可

**Review by SAKURA-CAT — CHANGES_REQUESTED** [review]

File: swanlab/data/system.py (hunk 1 above)
Comment (Feudalman) [reply swanlab/data/system.py:154]: 可以的

**Comment by SAKURA-CAT** [comment]
我们改变了存储结构，所以之前测试的依赖信息不存在了 @xiaolin199912

**Comment by xiaolin199912** [comment]
> 我们改变了存储结构，所以之前测试的依赖信息不存在了 @xiaolin199912

get

**Pull request merged**